iot-e2e-tests/common/src/main/java/com/microsoft/azure/sdk/iot/common/helpers/IntegrationTest.java

```diff
@@ -37,4 +40,27 @@ protected void finished(Description description)
     public Timeout timeout = new Timeout(E2E_TEST_TIMEOUT_MS);
 
     public static boolean isBasicTierHub;
+
+    //By default, run all tests. Even if env vars aren't set
+    public static boolean runIotHubTests = Boolean.parseBoolean(Tools.retrieveEnvironmentVariableValue("RUN_IOTHUB_TESTS", "true"));
+    public static boolean runProvisioningTests = Boolean.parseBoolean(Tools.retrieveEnvironmentVariableValue("RUN_PROVISIONING_TESTS", "true"));
+
+    public abstract boolean isProvisioningTest();
+    public abstract boolean isIotHubTest();
+
+    @Before
+    public void checkIfTestShouldBeRun()
+    {
+        if (isIotHubTest())
+        {
+            //Will skip the test if it is an iot hub test when told to skip iot hub tests
```

Comment:
Nit: this wording is a bit confusing, can be reworded into something along the lines of:
If it is an iot hub test, will skip the test when told to skip iot hub tests

Reply:
sure thing, I'll use your re-wording here and below